tests/microos/toolbox.pm

```diff
@@ -60,7 +60,7 @@ sub run {
     select_console 'root-console';
     $self->create_user;
 
-    my $toolbox_image_to_test = get_var('CONTAINER_IMAGE_TO_TEST');
+    my $toolbox_image_to_test = get_var('CONTAINER_IMAGE_TO_TEST_STANDALONE');
```

Comment:
I'm really not happy about his change. `CONTAINER_IMAGE_TO_TEST` is a descriptive and generic setting and use as such. If we need to test a system image, this setting is set to the URL of that system image. If we need to test a the toolbox image, this is set to the URL of that toolbox image.

What is the problem of having a context-sensitive but descriptive setting?

I feel this is a PR that we should discuss in a meeting.

I'm blocking it for now until we clarified the rationale and intention.

Reply:
ok, but it is a different test flow triggering. I can show you that it is a real project issue, a.t.m., if we use toolbox.pm module followed by other containers tests flows, in particular when calling `get_image_uri()`.